### ISSUE TITLE
no terminal emulator included

### ISSUE THREAD

**Issue by boogerlad** [michaelforney/oasis#1]
Why not use your st fork? Speaking of that, your st fork works fine for me on weston, but not sway(using wlc) This is what happens with `gdb st`

```

Program received signal SIGSEGV, Segmentation fault.
0x00007ffff7124396 in wl_proxy_marshal_constructor () from /usr/lib/libwayland-client.so.0

```

**Comment by michaelforney** [comment]
That's the plan.

Regarding your crash, that's pretty hard to debug with no info. Please build with debugging symbols and provide a full backtrace (preferably in an st issue). Running with WAYLAND_DEBUG=1 would also be helpful.

**Comment by michaelforney** [comment]
Fixed by fe210e3036fc6685e6b7fb93a66819fa8f6d0b4b.
